src/source/Sctp/Sctp.h

```diff
@@ -38,9 +38,14 @@ enum {
     DCEP_DATA_CHANNEL_OPEN = 0x03,
 };
 
-enum {
+typedef enum {
     DCEP_DATA_CHANNEL_RELIABLE = 0x00,
```

Comment:
Force a byte type in enum (not all compilers will respect). DCEP_DATA_CHANNEL_RELIABLE = (BYTE) 0x00, etc...

Reply:
Done.